data-generation/src/main/scala/thesis/SnapshotDelta.scala

```diff
@@ -27,10 +27,10 @@ abstract class TemporalGraph[VD: ClassTag, ED: ClassTag] extends Serializable {
 
 class SnapshotDelta(val graphs: MutableList[Snapshot],
                     val logs: RDD[LogTSV],
-                    val snapshotType: SnapshotIntervalType) extends TemporalGraph[Attributes, Attributes] { // extends Graph[VD, ED] {
+                    val snapshotType: SnapshotIntervalType) extends TemporalGraph[Attributes, (EdgeId, Attributes)] { // extends Graph[VD, ED] {
```

Comment:
```suggestion
                    val snapshotType: SnapshotIntervalType) extends TemporalGraph[Attributes, (EdgeId, Attributes)] { 
```
Fjerne kommentaren en gang for alle?